department-of-veterans-affairs/va.gov-cms
# VACMS-5848: Remove home crumb for Knowledge Base pages.

## Description
Closes #5848

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/125664601-fc81ceb6-2448-45b3-8690-a3e5d0257db5.png)

## QA steps
 - As a user who can view help pages, go to `/help/cms-knowledge-base` and visually verify `Home` isn't in breadcrumb trail.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
